gov/simulation/msgs.go

```diff
@@ -1,225 +0,0 @@
-package simulation
```

Comment:
any particular reason for removing this?

Reply:
It was full of `snyk` vulnerabilities detected. Once I started removing function after function, I figured out that the whole package was not used at all. It's kind of dead code, hence I removed it.